bin/Sherpa/cards/production/13TeV/higgs/Run.dat_ttbb_4FS_OpenLoops_13TeV

```diff
@@ -26,15 +26,14 @@
 
   %me generator settings
   ME_SIGNAL_GENERATOR Comix Amegic LOOPGEN;
-  OL_PREFIX=/cvmfs/cms.cern.ch/slc6_amd64_gcc530/external/openloops/1.3.1
   LOOPGEN:=OpenLoops;
 
   % settings relevant for 4-flavour scheme
   MASSIVE[5]=1
-  MASS[5]=4.75
+  MASS[5]=4.8
   PDF_LIBRARY LHAPDFSherpa
   #LHAPDF_GRID_PATH=./PDFsets
-  PDF_SET NNPDF30_nnlo_as_0118_nf_4
+  PDF_SET NNPDF31_nnlo_as_0118_nf_4
```

Comment:
please copy this datacard (don't remove the old one) to 
bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_ttbb_4FS_OpenLoops_13TeV